darts/models/forecasting/block_rnn_model.py

```diff
@@ -213,8 +213,15 @@ def __init__(
             Number of epochs to wait before evaluating the validation loss (if a validation
             ``TimeSeries`` is passed to the :func:`fit()` method).
         torch_device_str
-            Optionally, a string indicating the torch device to use. (default: "cuda:0" if a GPU
+            Optionally, a string indicating the torch device to use. (default: ``None``. Set "cuda:0" if a GPU
             is available, otherwise "cpu")
+
+            .. deprecated:: v0.17.0
+                ``torch_device_str`` has been deprecated in v0.17.0 and will be removed in a future version.
+                Instead, specify this with keys ``"accelerator", "gpus", "auto_select_gpus"`` in your
+                ``pl_trainer_kwargs`` dict. For more info, see here:
```

Comment:
Could you maybe write down which device will be used by default? 
Also, you could maybe add a small example here. e.g. "use: `gpus=-1` to use all available GPUs" or something along those lines. Or maybe using "auto"?